api/report_server.thrift

```diff
@@ -188,6 +197,9 @@ service codeCheckerDBAccess {
                          throws (1: shared.RequestFailed requestError),
 
   // PERMISSION: PRODUCT_ACCESS
+  RunHistoryDataList getRunHistory(1: i64 runId)
+                                   throws (1: shared.RequestFailed requestError),
+
   ReportData getReport(
```

Comment:
The comment line for what permission is needed to `getReport()` went missing as the new function was introduced above.